src/encoded/commands/load_access_keys.py

```diff
@@ -54,7 +56,12 @@ def generate_access_key(testapp, env, user_uuid, description):
     Returns:
         dict: access key contents with server
     """
-    server = get_beanstalk_real_url(env)
+    try:
```

Comment:
Do we really not have any manifest in-memory notion of whether we expect an ECS or EB url so that we can just make this an ordinary `if` and not a `try`?